.gitignore

```diff
@@ -168,3 +168,4 @@ config/**/*.gen.yaml
 config/**/*.gen.json
 
 *-installer*
+pkg/resources/resources.go
```

Comment:
@squakez 
Btw, simply adding this to `.gitignore` doesn't ignore future changes to this file. To completely eliminate it from repo, we need to remove it as well.

Reply:
True. I will fix it, thanks!

Reply:
I've made the change in ae815ddc01d98f4660c5d099f3f11ace3f346c9b, however, since then, the `validate` is failing. Maybe it required the presence of such a resource to be executed correctly. I'll revert the change to see what happens.

Reply:
I confirm, the commit in https://github.com/apache/camel-k/pull/3920 will fix that. It seems it is required, so, I will work to regenerate that.